test/e2e/entry.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2023 Antrea Authors
```

Comment:
The `fixtures.go` file was meant for functions such as these, but if you prefer a new file that's fine by me too

Reply:
Yes, that's what I was suggesting

Reply:
Done.